frontend/tests/ui/test_group.py

```diff
@@ -9,15 +9,81 @@
 
 from ui import util  # pylint: disable=no-name-in-module
 
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
 
 @util.annotate
-@util.no_javascript_errors()
-def test_create_group_with_identities(driver):
+@util.no_javascript_errors(path_whitelist=('/problem/mine/',))
```

Comment:
```suggestion
@util.no_javascript_errors()
```

hay que intentar no meter cosas al whitelist.

Reply:
Listo, ya se lo quité